src/core/store/notifications/getters.js

```diff
@@ -17,7 +19,7 @@ const txNotifications = function (state, _, rootState, rootGetters) {
   const filteredArray = state.notifications.filter(item => {
     if (
       item.from?.toLowerCase() === address?.toLowerCase() &&
-      item.type === 'OUT' &&
+      item.type.toLowerCase() === notificationTypes.out &&
```

Comment:
lets change the places where we set these types to import the notficatiion types, that way we dont have to do .toLowerCase

Reply:
after going through the PR I think you already switched it everywhere to import it from notificationTypes, you can safely take out .toLowerCase()

Reply:
sounds good!